steps/terraform.yaml

```diff
@@ -93,6 +96,11 @@ steps:
       filePath: $(System.DefaultWorkingDirectory)/cnp-azuredevops-libraries/scripts/build-resource-values.sh
       workingDirectory: $(System.DefaultWorkingDirectory)/$(buildRepoSuffix)
 
+  - template: ./set-terraform-varfile-env-var.yaml
+    parameters:
+      terraformVarFile: ${{ parameters.terraformVarFile }}
+      environment: ${{ parameters.environment }}
+
```

Comment:
this can go into existing `bash` script

Reply:
Removed the extra template in favour of using yaml if statements

Reply:
...even better 🙂